dal/dal.py

```diff
@@ -0,0 +1,94 @@
+from mongodb.mongodb_handler import (
+    update_item_status,
+    update_patron_items,
+    insert_document,
+    delete_document, delete_bill, update_bill, insert_bill
+)
+
+
+def update_item_dal(isbn, is_borrowed):
+    """
+    Update the status of a library item identified by its ISBN.
+
+    Args:
+        isbn (str): The ISBN of the library item.
+        is_borrowed (bool): True if the item is borrowed, False if it is returned.
+
+    """
+    update_item_status(isbn, is_borrowed)
+
+
+def update_patron_dal(patron_id, isbn, action, borrow_date=None):
```

Comment:
Like @fran2810 said, It seems that you are trying to create an abstraction for your DAL (which is good). But this is not the way to go about it. Read about ABC (abstract base class)